solarized-definitions.el

```diff
@@ -1,5 +1,9 @@
 (eval-when-compile
-  (require 'cl))
+  (require 'cl-lib)
```

Comment:
You actually have to remove this line, otherwise it’ll still break on older versions of Emacs without cl-lib.